src/agoradatatools/etl/load.py

```diff
@@ -160,3 +162,20 @@ def dict_to_json(df: dict, staging_path: str, filename: str) -> str:
     json.dump(df_as_dict, temp_json, cls=NumpyEncoder, indent=2)
     temp_json.close()
     return temp_json.name
+
+
+def list_to_json(df: List[Dict[str, Any]], staging_path: str, filename: str) -> str:
+    """Converts a list into a JSON file.
+
+    Args:
+        df (list): List to be converted to a JSON file
```

Comment:
Nit: Can you add to the comment that the list needs to be a list of Dicts? Also, what happens if the list is a DataFrame? Does it still work? Do we have a unit test for this function?

Reply:
We do not have a unit test for this function. I'll make one ASAP and include it in the PR.

Reply:
Before you do that, see my other comment in this file!